lua/jmod/cl_gui.lua

```diff
@@ -5,7 +5,8 @@ local YesMat=Material("icon16/accept.png")
 local NoMat=Material("icon16/cancel.png")
 local FavMat=Material("icon16/star.png")
 local SpecialIcons={
-	["geothermal"]=Material("ez_resource_icons/geothermal.png")
+	["geothermal"]=Material("ez_resource_icons/geothermal.png"),
+	["warning"]=Material("ez_misc_icons/warning.png")
```

Comment:
i think this doesn't need to be here

Reply:
You're right, I forgot about it when I switched to the display method we are using now.